synapse/handlers/__init__.py

```diff
@@ -12,36 +12,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
This file needs to stay so we can access this package, wasn't sure if I should lave the license info or not though or if it should just be a blank file?

Reply:
I think other `__init__` have copyright notices

Reply:
They seem to! 👍